src/datacrumbs/explorer/probe_explorer.cpp

```diff
@@ -286,9 +309,9 @@ std::vector<std::shared_ptr<Probe>> ProbeExplorer::extractProbes() {
         functionNames = std::move(filteredNames);
       }
     }
-    if (capture_probe->probe_type != ProbeType::CUSTOM) {
-      std::sort(functionNames.begin(), functionNames.end());
-    }
+    // if (capture_probe->probe_type != ProbeType::CUSTOM) {
+    //   std::sort(functionNames.begin(), functionNames.end());
+    // }
```

Comment:
Why was this commented? I would keep them sorted as it helps in checking functions and copying functions around.